spec/models/miq_widget_spec.rb

```diff
@@ -56,6 +56,33 @@
       '))
     end
 
+    describe "#sync_schedule" do
+      let(:schedule) do
+        filter = @widget_chart_vendor_and_guest_os.filter_for_schedule
+        FactoryBot.create(:miq_schedule, :filter => filter,
+                                           :resource_type => "MiqWidget", :name => @widget_chart_vendor_and_guest_os.name)
```

Comment:
Is this aligned with something?

Reply:
it was aligned in my environment which is showing inline hints :( , will fix